ddtrace/internal/serverless/mini_agent.py

```diff
@@ -1,29 +1,50 @@
 import os
 from subprocess import Popen
+import sys
 
-from . import in_gcp_function
 from ..compat import PYTHON_VERSION_INFO
 from ..logger import get_logger
+from ..serverless import in_azure_function_consumption_plan
+from ..serverless import in_gcp_function
 
 
 log = get_logger(__name__)
 
 
 def maybe_start_serverless_mini_agent():
-    if not in_gcp_function():
+    if not (in_gcp_function() or in_azure_function_consumption_plan()):
         return
 
-    try:
-        python_folder_name = "python%d.%d" % PYTHON_VERSION_INFO[:2]
-
-        rust_binary_path_default = (
-            "/layers/google.python.pip/pip/lib/%s/site-packages/"
-            "datadog-serverless-agent-linux-amd64/datadog-serverless-trace-mini-agent"
-        ) % python_folder_name
+    if sys.platform != "win32" and sys.platform != "linux":
+        log.error("Serverless Mini Agent is only supported on Windows and Linux.")
```

Comment:
We should make this error message actionable. Maybe include a configuration that be set to disable this feature?

Reply:
this log line  is a bit redundant since this functions shouldn't be called outside of windows + linux anyways, what do you think about removing this entire if block?